File: Shank/benchMark/ManderboltSet/Manderbolt.py
def manderobolt():
    px = 100
    py = 100
    x0 = -1.0
    y0 = 0.47
    for i in range(px):
        x = 0
        y = 0
        for i2 in range(py):
            max_it = 100
            tmp = x * x + y * y
            while tmp <= 4.0:
                for idx in range(max_it):
                    xtemp = x * x - y * y + x0
                    y = 2.0 * x * y + y0
                    x = xtemp
                    tmp = x * x + y * y
manderobolt()
print("ran")

File: Shank/benchMark/ManderboltSet/Manderbolt.c

int main()
{
    int px = 100;
    int py = 100;
    double x0 = 0.47;
    double y0 = -1.0;
    for (int ix = 0; ix < px; ix++)
    {
        long double x = 0;
        long double y = 0;
        for (int iy = 0; iy < py; iy++)
        {
            long max_it = 100;
            long double tmp = 0;
            tmp = x * x + y * y;
            while (tmp <= 4.0)
            {
                for (int idx = 0; idx < max_it; idx++)
                {
                    long double xtemp = x * x - y * y + x0;
                    y = 2.0 * x * y + y0;
                    x = xtemp;
                    tmp = x * x + y * y;

                }
                
            }
        }
    }
}
